nvflare/private/fed/client/client_runner.py

```diff
@@ -76,6 +76,14 @@ def add_component(self, comp_id: str, component: object):
             self.handlers.append(component)
 
 
+class TaskInfo:
```

Comment:
The TaskInfo needs to built into the Task class. They are directly tied together.

Reply:
Removed TaskInfo. Was used for keeping and managing task instances. Now it's simplified.